app/views/like/_like.html.erb

```diff
@@ -27,70 +27,114 @@
       </span>
     <% end %>
   <% end %>
-
-  <span data-toggle="tooltip" data-placement="top" title="Tools" class="btn btn-outline-secondary btn-circle btn-sm" rel="popover" data-placement="left" data-html="true" style="overflow: auto; max-height: 500px" data-content="
-
-    <p><b>Users who liked this</b></p>
-    <% node.likers.each do |user| %>
-      <i class='fa fa-star-o'></i> <a href='/profile/<%= user.username %>/'><%= user.username %></a><br />
-    <% end %>
-    <% if node.likers.length == 0 %><i>None</i><% end %>
-    <% if node.type == 'note' && current_user && node.uid != current_user.uid%>
-    <hr style='margin-bottom:3px;' />
-    <p><b>Award a Barnstar:</b></p>
-    <form class='form' action='/barnstar/give'>
-    <select name='star' class='form-control'>
-        <option value='basic'>The basic barnstar</option>
-        <option value='photo-documentation'>The photo documentation barnstar</option>
-        <option value='video-documentation'>The video documentation barnstar</option>
-        <option value='watchdog'>The watchdog barnstar</option>
-        <option value='empiricism'>The empiricism barnstar</option>
-        <option value='excessive-enthusiasm'>The enthusiasm barnstar</option>
-    </select><br />
-    <input type='hidden' name='nid' value='<%= node.id %>'/>
-    <input type='submit' class='btn btn-primary' value='Give'/> &nbsp; <a _target='blank' href='/wiki/barnstars'>Learn more</a>
-    </form>
+  <div class="dropdowna">
+  <li id="ellipsis_sidebar" class="btn btn-outline-secondary btn-circle btn-sm" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      <span class="ml-2 ff fa fa-ellipsis-h" style="font-size: 1em;"></span>
+    </li>
+  <ul class="dropdown-menu">
+    <li class="dropdown-item">
+      <a href="/n/<%= @node.id %>"><i class="fa fa-link"></i></a> <span class="d-none d-xl-inline"><a href="/n/<%= @node.id %>">#<%= @node.id %></a></span>
```

Comment:
Oh! Aha! You need to be using `node` here as it's a local variable in this template! For all instances of `@node` in this file. That should fix it!

Reply:
Ohhh!!! I'll fix it.
Thanks Jeff